plugins/inputs/statsd/datadog.go

```diff
@@ -0,0 +1,187 @@
+package statsd
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	tmetric "github.com/influxdata/telegraf/metric"
```

Comment:
Is this alias needed?

Reply:
Yes, because there is a `metric` type used everywhere in this package, that collides with it.

Reply:
I see, over in statsd.go